### PR TITLE
Remove `/opt/ghc` initially

### PR DESCRIPTION
**Pull request by jakirkham**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: .azure-pipelines/azure-pipelines-linux.yml [hunk 1]
```diff
@@ -32,6 +32,11 @@ jobs:
   timeoutInMinutes: 360
 
   steps:
+  - script: |
+        rm -rf /opt/ghc
+        df -h
+    displayName: Manage disk space (rm /opt/ghc)
```
Comment (beckermr) [inline .azure-pipelines/azure-pipelines-linux.yml:38]: This works too.

Comment (jakirkham) [reply .azure-pipelines/azure-pipelines-linux.yml:38]: Thanks for checking 😄

Comment (jakirkham) [reply .azure-pipelines/azure-pipelines-linux.yml:38]: Submitted upstream in PR ( https://github.com/conda-forge/conda-smithy/pull/1345 ).

Comment (h-vetinari) [reply .azure-pipelines/azure-pipelines-linux.yml:38]: Thanks @jakirkham. I'm thinking we can probably close this one, and just rerender #8 after the smithy-PR lands.

**Pull request closed without merging**